daprovider/das/dasRpcClient.go

```diff
@@ -124,7 +123,7 @@ func (c *DASRPCClient) legacyStore(ctx context.Context, message []byte, timeout
 	// #nosec G115
 	log.Trace("das.DASRPCClient.Store(...)", "message", pretty.FirstFewBytes(message), "timeout", time.Unix(int64(timeout), 0), "this", *c)
 
-	reqSig, err := applyDasSigner(c.signer, message, timeout)
+	reqSig, err := c.signer(crypto.Keccak256(data_streaming.FlattenDataForSigning(message, timeout)))
```

Comment:
Doesn't this change the signature format and break backwards compatibility? How would we be able to coordinate rolling this out given that the client and servers are typically run by different organizations, multiplied by however many orbit deployments there are?